app/src/main/java/com/twilio/conversations/app/common/Injector.kt

```diff
@@ -6,9 +6,21 @@ import androidx.annotation.RestrictTo
 import androidx.annotation.RestrictTo.Scope
 import com.twilio.conversations.app.data.ConversationsClientWrapper
 import com.twilio.conversations.app.data.CredentialStorage
-import com.twilio.conversations.app.manager.*
+import com.twilio.conversations.app.manager.ConversationListManagerImpl
```

Comment:
Just keep it as * here and below.

Reply:
Lets keep it single-line style. Otherwise standalone kotlin functions imported from third-party libs could conflict with our own